### PR TITLE
[dev] upload workers sites bindings for edge dev

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
before we weren't handling any workers sites stuff for edge dev - now we do

### CODE REVIEW

File: src/commands/dev/edge/setup.rs [hunk 1]
```diff
@@ -31,6 +50,14 @@ pub(super) fn upload(
         .send()?
         .error_for_status()?;
 
+    if !to_delete.is_empty() {
```
Comment (jwheels) [inline src/commands/dev/edge/setup.rs:53]: Just double checking... this won't cause us to delete any production files, right?

Comment (ashleymichal) [reply src/commands/dev/edge/setup.rs:53]: that really depends on whether they put the same namespace in for preview_id.

Comment (EverlastingBugstopper) [reply src/commands/dev/edge/setup.rs:53]: yup - the namespace returned here is the one they have defined for their preview sesssions. we dont stop them from having production namespace in preview, but we print a warning out when they're the same that they are in dangerous territory.

Comment (jwheels) [reply src/commands/dev/edge/setup.rs:53]: cool, sounds good!

**Pull request merged**